plugins/uefi-dbx/fu-uefi-dbx-plugin.c

```diff
@@ -108,6 +111,15 @@ fu_uefi_dbx_plugin_constructed(GObject *obj)
 		/* TODO figure out non-snap scenarios */
 		g_info("snapd integration outside of snap is not supported");
 	}
+
+	/* ensure that an ESP was found */
+	esp = fu_context_get_default_esp(fu_plugin_get_context(plugin), &error_udisks2);
```

Comment:
does esp need to be unref'ed?

Reply:
Great catch.  will fix when I figure out what to do about CI.